application/config/database.php.copy

```diff
@@ -78,7 +78,7 @@ $db['default'] = array(
 	'hostname' => 'localhost',
 	'username' => 'root',
 	'password' => '',
-	'database' => '',
+	'database' => 'mckie',
```

Comment:
You dont have to edit this file anymore. This is just for reference on how to create database.php file